.asf.yaml

```diff
@@ -24,11 +24,16 @@ github:
     - universal-resource-scheduler
     - apache-yarn
     - kubernetes
+  notifications:
+    commits:      isseus@yunikorn.apache.org
```

Comment:
https://lists.apache.org/list.html?issues@yunikorn.apache.org is work, so I think that 
```
isseus@yunikorn.apache.org -> issues@yunikorn.apache.org
```